src/app/main/component/shared/components/events-list-item/events-list-item.component.ts

```diff
@@ -172,15 +173,15 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
         this.btnStyle = this.styleBtn.secondary;
         this.nameBtn = this.btnName.cancel;
         break;
-      case !isSubscribed && isRelevant && !this.isOwner && !isAdmin && !isUnauthorized:
+      case !isSubscribed && isRelevant && !this.isOwner && !isAdmin:
         this.btnStyle = this.styleBtn.primary;
         this.nameBtn = this.btnName.join;
         break;
       case isSubscribed && !isRelevant && !this.isOwner:
         this.btnStyle = this.styleBtn.primary;
         this.nameBtn = this.btnName.rate;
         break;
-      case (!isSubscribed && !isRelevant && !this.isOwner) || isUnauthorized:
```

Comment:
please remove the isUnauthorized variable from the method

Reply:
The variable has been removed.